app/views/interactive_pages/edit.html.haml

```diff
@@ -87,17 +87,28 @@
         .questions
           %h2 Info/Assess
           = form_tag add_embeddable_activity_page_path(@activity, @page), :class => 'embeddables-form' do
-            = embeddable_selector
+            .select_container
+              = embeddable_selector
             = hidden_field_tag :approved_script_id
             = hidden_field_tag :component_label
             = submit_tag 'Add Embeddable', {id: 'add_embeddable_button'}
-          .embeddables_list.sortable_embeddables
+          .embeddables_list.sortable_embeddables.accordion_embeddables
             - @page.section_embeddables(nil).each do |e|
               - if e.show_in_edit?
                 - tracked = e.respond_to?(:tracked_question) && e.tracked_question
                 - tracked_class = tracked ? 'tracked' : ''
                 .authorable{:id => "embeddable_#{e.id}.#{e.class.to_s}", :class => tracked_class}
-                  = render :partial => "#{e.class.name.underscore.pluralize}/author", :locals => { :embeddable => e, :page => @page, :allow_hide => true }
+                  %h3
+                    - if e.respond_to? :name
+                      - if !e.name.nil? && !e.name.empty?
+                        #{e.name}
+                      - else
+                        #{e.class.human_description.titleize}
```

Comment:
This is where you'd use `model_name.human` I'm not sure if you'll need titleize in this case or not.

Reply:
It doesn't matter that much, but that code `window.location.search.match('edit_')` isn't looking for a match in the full URL. It's looking for a match in `window.location.search` which is only the query string part of the URL. Since the edit parameter name varies depending on which assessment type you add, I thought that was the simplest way to just check that an item of any type had been added. I do understand it's not ideal, but the edit param names vary according to which type of assessment item you add and I was wary of adding something to check all possible edit param name possibilities.

Anyway, I think the code you pointed to in _embedded_editor_links.html.haml in a previous comment is a better example to follow though. I'll do something like that instead.

Also FWIW: When you edit an existing item, there isn't a param added to the URL. It's only when you add a new item. Maybe the param should be added when you edit something though?